nmstatectl/nmstatectl.py

```diff
@@ -127,24 +127,35 @@ def show(args):
 
 
 def apply(args):
-    # read from file when it is not '-' unless '-' actually exists
-    if args.file and (args.file != '-' or os.path.isfile(args.file)):
-        with open(args.file) as statefile:
-            statedata = statefile.read()
-    elif not sys.stdin.isatty() or args.file == '-':
+    if args.file:
+        for statefile in args.file:
+            if statefile == '-' and not os.path.isfile(statefile):
+                statedata = sys.stdin.read()
+
```

Comment:
nit: No need for this newline

Reply:
IMHO it matches the other newlines in this method.

Reply:
A newline after ‘else’ is very strange.

Reply:
I only saw newlines before the else. I removed them now, too. I really wish a tool would just take care of this formatting so I do not need to waste brain cycles on this. ;-)